dev-python/wurlitzer/wurlitzer-2.0.0.ebuild

```diff
@@ -0,0 +1,18 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+
+inherit distutils-r1
+
+DESCRIPTION="Capture C-level stdout/stderr in Python "
+HOMEPAGE="https://github.com/minrk/wurlitzer"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+distutils_enable_tests pytest
```

Comment:
Test fail.

================================ test session starts =================================
platform linux -- Python 3.6.9, pytest-4.6.8, py-1.8.0, pluggy-0.13.1 -- /usr/bin/python3.6
cachedir: .pytest_cache
rootdir: /var/tmp/portage/dev-python/wurlitzer-2.0.0/work/wurlitzer-2.0.0
collecting ... collected 0 items

Reply:
Removed the tests from wurlitzer and spyder-kernels.

Spyder-kernels lists the following dependencies for tests:
```
DEPEND="test? ( dev-python/numpy[${PYTHON_USEDEP}]
			dev-python/pandas[${PYTHON_USEDEP}]
			sci-libs/scipy[${PYTHON_USEDEP}]
			dev-python/flaky[${PYTHON_USEDEP}]
			dev-python/mock[${PYTHON_USEDEP}]
			dev-python/cython[${PYTHON_USEDEP}]
			dev-python/matplotlib[${PYTHON_USEDEP}]
			dev-python/xarray[${PYTHON_USEDEP}] )"
```
But I can't seem to get it to work....
It keeps telling me that no tests were run.
Also there aren't any tests in the spyder-kernels-{0.2.4, 1.0.1} ebuilds, so I just removed them.